deploy-tool/cert/cert.go

```diff
@@ -71,6 +67,26 @@ func setStorageDriver(deployConfig config.DeployConfig, acme *certmagic.Config)
 	return nil
 }
 
+// IssueCertificate asks the Let's Encrypt ACMEApiEndpoint to validate and sign a certificate for a site using the DNS01 strategy
+func (certManager *CertificateManager) IssueCertificate(site string, privKey *ecdsa.PrivateKey) (string, error) {
```

Comment:
It's a bit of a shame that this returns `string` because it's hard to guess what that string represents. I think in this case it's the PEM certificate chain. We could define a type alias (not certain if that's very idiomatic in Go) but at least let's document what the returned value is. Also nit: wrap long comment lines.

Reply:
If you mean something like `type CertificatePEM string`, that's very clean Go practice, from what I've seen.